docs/source/pretrained_models.rst

```diff
@@ -10,3 +10,17 @@ For convenience, the following pre-trained models are ready-to-use:
 - `findcord_tumor <https://github.com/ivadomed/findcord_tumor/archive/r20200621.zip>`_: Cord localisation model, trained on T2-weighted images with tumor.
 - `model_find_disc_t1 <https://github.com/ivadomed/model_find_disc_t1/archive/r20201013.zip>`_: Intervertebral disc detection model trained on T1-weighted images.
 - `model_find_disc_t2 <https://github.com/ivadomed/model_find_disc_t2/archive/r20200928.zip>`_: Intervertebral disc detection model trained on T2-weighted images.
+
+Packaged model format
+=====================
+Each folder contains a model (.pt or .onnx) with its corresponding configuration file. The packaged model is
+automatically generated during training. The folder containing the packaged model will be saved at the path specified by
+the key "log_directory" in the configuration file. The packaged model, the configuration file, and the model file will
```

Comment:
hum... when i tried a dummy training (with `ivadomed -c blabla.json`), i had to manually rename `best_model.py` and `config_file.json` to match the folder name:

![image](https://user-images.githubusercontent.com/2482071/104250140-f7f81880-543a-11eb-893a-f520039cea5a.png)

Reply:
You do not need to: they are in "seg_sc_t1-t2-t2s-mt" --> which is the "model_name", cf doc added by @andreanne-lemay about `model_name`

Reply:
oh gosh! something else i missed. How about calling it: `packaged_model` instead of `seg_sc_t1-t2-t2s-mt`? We expect users to change the name anyway (to fit their conventions, which might be different than [our conventions](https://github.com/ivadomed/ivadomed/wiki/repositories#models)), and at least `packaged_model` has the merit of being more explicit (ie: this is the packaged model).

Reply:
@jcohenadad : Are you suggesting to change this [line](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/config/config.json#L5) to:
```
    "model_name": "packaged_model",
```
?

Sounds good. Note: if we go that way, we will need to change the tutorials (easy peasy), eg [here](https://ivadomed.org/en/latest/tutorials/one_class_segmentation_2d_unet.html#evaluate-model).

Reply:
or `"my_model"`, see [comment](https://github.com/ivadomed/ivadomed/pull/625#discussion_r555468959).

Reply:
yup, that's what i'm suggesting. Or, maybe better, we go with "my_model", as this is what I've used as an example in my recent comment https://github.com/ivadomed/ivadomed/pull/625/files#r555468959

Reply:
Sounds good to me